docker-compose.yml

```diff
@@ -53,6 +54,7 @@ services:
       args:
         URL: "https://github.com/algorand/indexer"
         BRANCH: "master"
+        # Is there a reason why we're still pinning to a particular commit?
```

Comment:
@winder  - I think you understand the context pretty well here. Do you think we should still be pinning to specific commits? Could you have a look at the other comments as well?

Reply:
It's been a long time since I looked at this. Do you have a recommendation for how this could be improved?

If I remember correctly, versions are pinned strategically to prevent new tests/features from breaking old releases. For example, if an endpoint were deprecated or a new field was added. i.e. `indexer-221-2` is something like version 2.2.1.

If we are going to improve this framework, I think we should push to get production docker containers.

@algojohnlee are we going to have production docker containers soon? [It would be great if we could use this as a solid foundation.](https://hub.docker.com/r/wwinder/algod)

Reply:
@tzaffi @winder Reading the thread it's unclear to me if the `indexer-221-X` images can be removed.  What's the verdict?

Provided it's _safe_, removing these images provides an immediate build time reduction.  Obviously, published containers are a welcomed future improvement.

Reply:
I'm happy to open another PR that aims at understanding which tests are impacted and -with support- removes such tests. Ok?